website/docs/guides/mesh-qs.md

```diff
@@ -0,0 +1,575 @@
+        - name: orders
+          description: One record per order. Includes cancelled and deleted orders.
+  ```
+  </File>
+
+3. Create a `models/staging/stg_customers.sql` file to select from the `customers` table in the `jaffle_shop` source.
+
+  <File name='models/staging/stg_customers.sql'>
+
+  ```sql
+  select
+      id as customer_id,
+      first_name,
+      last_name
+
+  from {{ source('jaffle_shop', 'customers') }}
+  ```
+  </File>
+
+4. Create a `models/staging/stg_orders.sql` file to select from the `orders` table in the `jaffle_shop` source.
+
+  <File name='models/staging/stg_orders.sql'>
+
+  ```sql
+  select
+      id as order_id,
+      user_id as customer_id,
+      order_date,
+      status
+
+  from {{ source('jaffle_shop', 'orders') }}
+  ```
+  </File>
+
+5. Create a `models/core/fct_orders.sql` file to build a fact table with customer and order details.
+
+  <File name='models/core/fct_orders.sql'>
+
+  ```sql
+  with customers as (
+      select * 
+      from {{ ref('stg_customers') }}
+  ),
+
+  orders as (
+      select * 
+      from {{ ref('stg_orders') }}
+  ),
+
+  customer_orders as (
+      select
+          customer_id,
+          min(order_date) as first_order_date
+      from orders
+      group by customer_id
+  ),
+
+  final as (
+      select
+          o.order_id,
+          o.order_date,
+          o.status,
+          c.customer_id,
+          c.first_name,
+          c.last_name,
+          co.first_order_date,
+          -- Note that we've used a macro for this so that the appropriate DATEDIFF syntax is used for each respective data platform
+          {{ dbt_utils.datediff('first_order_date', 'order_date', 'day') }} as days_as_customer_at_purchase
+      from orders o
+      left join customers c using (customer_id)
+      left join customer_orders co using (customer_id)
+  )
+
+  select * from final
+  ```
+  </File>
+
+6. Execute `dbt build` command.
+
+Before a downstream team can leverage assets from this foundational project, you need to first:
+- [Create and define](/docs/collaborate/govern/model-access) at least one model as “public”
+- Run a [deployment job](/docs/deploy/deploy-jobs) successfully
+  - Note, Enable the **Generate docs on run** toggle for this job to update the dbt Explorer. Once run, you can link dbt Explorer to a deployment job on the **Project Settings** page (Refer to [artifacts](/docs/deploy/artifacts) for more information).
+
+## Define a public model and run first job
+
+In the previous section, you've arranged your basic building blocks, now let's integrate dbt Mesh. Although the Finance team requires the `fct_orders` model for analyzing payment trends, other models, particularly those in the staging layer used for data cleansing and joining, are not needed by downstream teams.
+
+To make `fct_orders` publicly available, add an `access: public` clause to the relevant YAML file. Add and save the following file to the project:
```

Comment:
this means they have to create a 'models/core/core.yml' file in the Jaffle | Data Analytics project, right?

Reply:
yup! in line 211 I noted they'll add and save the 'models/core/core.yml' file